tests/frameworks/gtest/include/gtest/internal/gtest-param-util.h

```diff
@@ -598,6 +599,16 @@ class ParameterizedTestCaseInfo : public ParameterizedTestCaseInfoBase {
     }  // for test_it
   }  // RegisterTests
 
+  std::vector<std::pair<std::string, TestMetaFactoryBase<typename TestCase::ParamType>*>> GetTestMetaFactories() {
+    std::vector<std::pair<std::string, TestMetaFactoryBase<typename TestCase::ParamType>*>> toReturn;
+    for (typename TestInfoContainer::iterator test_it = tests_.begin(); test_it != tests_.end(); ++test_it) {
```

Comment:
I'll try but I think I did that and it was complaining about needing the typename stuff. Maybe auto will work though.